composer.lock

```diff
@@ -0,0 +1,1827 @@
+{
```

Comment:
We'll want to keep this out of here. We want to commit `composer.lock` on projects, but not on templates/generators.

Reply:
Cool, thanks for explaining.

Reply:
Okay, composer.lock is gone